grid_map_common/CMakeLists.txt

```diff
@@ -0,0 +1,14 @@
+cmake_minimum_required(VERSION 3.5)
```

Comment:
How do you feel about this common package @maximilianwulf? We have a common pkg in nav2 to reduce the cmake boilerplate but open to do whatever you think is best for this project.